packages/mux-uploader/README.md

```diff
@@ -97,42 +97,44 @@ This also means you can implement your own drag and drop (or other) components f
-| `formatProgress` | `function` | A function that accepts numeric percent and is expected to return a string. Allows for customizing how the progress should be rendered. | A function the yields a percent progress string |
+| Property         | Type       | Description                                                                                                                                                                                                                                                                                        | Default                                                          |
+| ---------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `formatProgress` | `function` | A function that accepts numeric percent and is expected to return a string. Allows for customizing how the progress should be rendered - whether you want to display only the number or a sentence with the number etc. i.e. `formatProgress={(percent: number) => `${percent} percent uploaded`}` | A function that yields only the percent as a string i.e. `"60%"` |
 
 ### Events
 
 #### `<mux-uploader>`
 
-`<mux-uploader>` has a handful of events to monitor uploading state.
+`<mux-uploader>` has a handful of events - using [Upchunk](https://github.com/muxinc/upchunk/blob/master/README.md) under the hood - to monitor uploading state.
 
-| Event      | Description                                                              |
-| ---------- | ------------------------------------------------------------------------ |
-| `error`    | Fired when an error occurs in the chunked upload process.                |
-| `progress` | Fired whenever a chunk of the file has successfully completed uploading. |
-| `success`  | Fired when the entire file has successfully completed uploading.         |
+| Event          | Description                                                                                                                                                        |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `attempt`      | Fired immediately before a chunk upload is attempted.                                                                                                              |
+| `chunkSuccess` | Fired when an indvidual chunk is successfully uploaded. Sample response: `{ detail: { chunk: Integer, attempts: Integer, response: XhrResponse } }`                |
```

Comment:
sorry, I didn't review before the merge.

blocking(release): `chunkSuccess` should be lowercase `chunksuccess` or `chunk-success` 

nit(non-blocking): `attempt` I've rarely seen this word written in code, it feels a bit strange maybe because of the former but wouldn't something more like the HTMLVideoElement event `loadstart` work here. 
maybe `uploadstart`?